ngraph/test/runtime/opset0_downgrade.hpp

```diff
@@ -19,6 +19,8 @@
 #include "backend_visibility.hpp"
 #include "ngraph/pass/pass.hpp"
 
+NGRAPH_SUPPRESS_DEPRECATED_START
```

Comment:
Doesn't this call require a matching `NGRAPH_SUPPRESS_DEPRECATED_END`? The same goes for the next 2 files in this PR.

Reply:
No it means that it works for whole file.

Reply:
Yes, but in hpp we must use start / end, otherwise the suppresion is used for all cpp files which include this hpp

Reply:
Oh, sure, will fix